packages/client/src/containers/LandingPage/LandingPage.Container.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Testimonials from '../../components/Login/Testimonials';
+import logo from './logo.svg';
+import { Typography, TextField, Button } from '@mui/material';
+import './LandingPage.Style.css';
+
+const LandingPageContainer = () => {
+  return (
+    <div className="landing-page">
+      <div className="testimonial-container">
+        <Typography className="text-field" variant="h4" color="white">
```

Comment:
Please move the `color="white"` to css. It is difficult to maintain styles in the `html` section

Reply:
@ddobby94 it is a prop for appearance for a MUI component. 
Is it common to still style it with custom CSS in this case?